0023-bilateral-transfer-protocol/0023-bilateral-transfer-protocol.md

```diff
@@ -465,12 +300,9 @@ Transfer ::= SEQUENCE {
 }
 ```
 
-`Transfer` is used to send money unconditionally on the bilateral ledger.
-Unlike a `Prepare`, `Transfer` is assumed to move funds immediately, without
-requiring a subsequent `Fulfill` call.
-
-Data to secure this payment (such as payment channel claims), can be attached
-under the protocol data.
+`Transfer` is used to send payment channel claims to the other connector.
```

Comment:
True, but I wonder if this is too specific. Could `Transfer`s be used to send - for instance - a PayPal payment ID as well to trigger processing of some other form of outside settlement?